src/Controller/Admin/OrderOverview.php

```diff
@@ -119,6 +121,32 @@ public function render()
         return parent::render();
     }
 
+    public function getAmazonAPIOrderStatus(): string
+    {
+        if (is_null($this->captureStatus)) {
+            $this->captureStatus = '';
+            $orderId = $this->getEditObjectId();
+            if ($orderId !== '-1') {
+                $lang = Registry::getLang();
+                $repository = oxNew(LogRepository::class);
+                $order = oxNew(Order::class);
+                $order->load($orderId);
+                $logMessage = $repository->findLogMessageForOrderId($orderId);
+                $chargePermissionId = $logMessage[0]['OSC_AMAZON_CHARGE_PERMISSION_ID'];
```

Comment:
to avoid key not found error:

`$chargePermissionId = $logMessage[0]['OSC_AMAZON_CHARGE_PERMISSION_ID'] ?? null;`